.cbmc-batch/stubs/aws_string_destroy_override.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <aws/common/string.h>
+
+/**
+ * Saves the function pointer dereferences when doing the delloc. Otherwise the same as the
```

Comment:
s/delloc/dealloc/, and I don't really understand "Saves" here - more context should be added.

Reply:
Done